src/components/Carousel/Carousel.tsx

```diff
@@ -139,7 +139,7 @@ export const Carousel = ({
             {renderSlides}
           </Box>
         </Box>
-        {(showDots || showArrows) && scrollSnaps.length > 1 && (
+        {(showDots || showArrows) && (
```

Comment:
q: what does dis do?

Reply:
It always shows the dot bar (pagination bar) even if there are no element yet, if we don't show it, we will have some layout shifts because the height will come after the first load, this ensure we always have the place for this element so it doesn't cause layout shift